R/path_from.R

```diff
@@ -0,0 +1,12 @@
+path_from <- function(from = NULL, ...) {
+  if (is.null(from)) return(here::here(...))
+  here::here(from, ...)
+}
+
+raw_inputs <- function() {
+  list("working_dir", "20_Raw_Inputs")
```

Comment:
My intent is for this to be the only place that knows the exact path to this weird directory.